code/ui/components/src/tooltip/TooltipLinkList.stories.tsx

```diff
@@ -61,18 +45,160 @@ export default {
+        href: 'http://google.com',
+      },
+      {
+        id: '2',
+        title: 'Link 2',
+        center: 'This is an addition description',
+        href: 'http://google.com',
+      },
+    ],
     LinkWrapper: StoryLinkWrapper,
   },
-};
+} satisfies Story;
+
+export const ActiveWithoutAnyIcons = {
+  args: {
+    links: [
+      {
+        id: '1',
+        title: 'Link 1',
+        active: true,
+        center: 'This is an addition description',
+        href: 'http://google.com',
+      },
+      {
+        id: '2',
+        title: 'Link 2',
+        center: 'This is an addition description',
+        href: 'http://google.com',
+      },
+    ],
+    LinkWrapper: StoryLinkWrapper,
+  },
+} satisfies Story;
+
+export const ActiveWithSeparateIcon = {
+  args: {
+    links: [
+      {
+        id: '1',
+        title: 'Link 1',
+        icon: 'link',
+        center: 'This is an addition description',
+        href: 'http://google.com',
+      },
+      {
+        id: '2',
+        title: 'Link 2',
+        active: true,
+        center: 'This is an addition description',
+        href: 'http://google.com',
+      },
+    ],
+    LinkWrapper: StoryLinkWrapper,
+  },
+} satisfies Story;
+
+export const ActiveAndIcon = {
+  args: {
+    links: [
+      {
+        id: '1',
+        title: 'Link 1',
+        active: true,
+        icon: 'link',
+        center: 'This is an addition description',
+        href: 'http://google.com',
+      },
+      {
+        id: '2',
+        title: 'Link 2',
+        center: 'This is an addition description',
+        href: 'http://google.com',
+      },
+    ],
+    LinkWrapper: StoryLinkWrapper,
+  },
+} satisfies Story;
+
+export const WithIllustration = {
+  args: {
+    links: [
+      {
+        id: '1',
+        title: 'Link 1',
+        active: true,
+        icon: 'link',
+        right: <img src={ellipseUrl} width="16" height="16" alt="ellipse" />,
+        center: 'This is an addition description',
+        href: 'http://google.com',
+      },
+      {
+        id: '2',
+        title: 'Link 2',
+        center: 'This is an addition description',
+        right: <img src={ellipseUrl} width="16" height="16" alt="ellipse" />,
+        href: 'http://google.com',
+      },
+    ],
+    LinkWrapper: StoryLinkWrapper,
+  },
+} satisfies Story;
+
+export const WithCustomIcon = {
+  args: {
+    links: [
+      {
+        id: '1',
+        title: 'Link 1',
+        active: true,
+        icon: <Icons icon="linux" />,
+        right: <img src={ellipseUrl} width="16" height="16" alt="ellipse" />,
+        center: 'This is an addition description',
+        href: 'http://google.com',
+      },
+      {
+        id: '2',
+        title: 'Link 2',
+        center: 'This is an addition description',
+        right: <img src={ellipseUrl} width="16" height="16" alt="ellipse" />,
+        href: 'http://google.com',
+      },
+    ],
+    LinkWrapper: StoryLinkWrapper,
+  },
+} satisfies Story;
```

Comment:
@kasperpeulen WDYT about using satisfies for stories rather than `MyStory: Story = {}` ?